lib/msal-browser/src/cache/BrowserCacheManager.ts

```diff
@@ -288,6 +295,7 @@ export class BrowserCacheManager extends CacheManager {
      * @param appMetadataKey
      */
     getAppMetadata(appMetadataKey: string): AppMetadataEntity | null {
+        this.logger.verbose("BrowserCacheManager.getAppMetadata called");
         const value = this.getItem(appMetadataKey);
         if (!value) {
             return null;
```

Comment:
In each of the getters should we also log whether or not we successfully found something in the cache?

Reply:
+1

Reply:
Makes sense. I added cache hit logs, and it makes the logs a lot longer. If we discover it creates too much noise, I'm happy to remove them later.